model_zoo/ECG2AF/deployment/v1/processing_image/prepare.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+
+import h5py
+import numpy as np
+import pandas as pd
+import smart_open
+
+ECG_REST_LEADS = {
```

Comment:
This is ok for now, but since these constants are already defined in `ml4h/defines.py` it would be better to include `ml4h` and import from there. But that would require making the docker image much bigger.